pkg/catalogsourceconfig/deleted.go

```diff
@@ -82,11 +89,13 @@ func (r *deletedReconciler) deleteCreatedResources(ctx context.Context, name, na
 		CscOwnerNamespaceLabel: namespace,
 	}
 	labelSelector := labels.SelectorFromSet(labelMap)
-	options := &client.ListOptions{LabelSelector: labelSelector}
+	catalogSourceOptions := &client.ListOptions{LabelSelector: labelSelector}
```

Comment:
Can't we also restrict this to the namespace where we know the `CatalogSource` is present by looking at the `targetNamespace`?

Reply:
That is true. It also means (and this is true for all of the resources we are doing this to) if they are moved to a different namespace without us being made aware of it, we won't be able to delete them. We won't error because we are using List() instead of Get(), but we will miss them.

@aravindhp @tkashem thoughts? What's the better scenario?

Reply:
If resources got moved to other namespaces without us being aware, we are going to have other issues. My vote is for that scenario to be tackled when we starting watching our resources.

Reply:
@aravindhp updated.